frontend/src/components/grantaccess/GrantAccess.vue

```diff
@@ -267,53 +312,6 @@ const composeAndPushNotificationMessages = (
     }
     return '';
 };
-
-const handleSubmit = async () => {
-    const successForestClientIdList: string[] = [];
-    const warningForestClientIdList: string[] = [];
-    const errorForestClientIdList: string[] = [];
-
-    do {
-        const item = forestClientData.value.pop();
-        const data = toRequestPayload(formData.value, item);
-
-        await userRoleAssignmentApi
-            .createUserRoleAssignment(data)
-            .then(() => {
-                successForestClientIdList.push(
-                    item?.forest_client_number ? item.forest_client_number : ''
-                );
-            })
-            .catch((error) => {
-                if (error.response?.status === 409) {
-                    warningForestClientIdList.push(
-                        item?.forest_client_number
-                            ? item?.forest_client_number
-                            : ''
-                    );
-                } else {
-                    errorForestClientIdList.push(
-                        item?.forest_client_number
-                            ? item.forest_client_number
-                            : ''
-                    );
-                }
-            });
-    } while (forestClientData.value.length > 0);
-
-    composeAndPushNotificationMessages(
-        successForestClientIdList,
-        warningForestClientIdList,
-        errorForestClientIdList
-    );
-
-    router.push('/dashboard');
-};
-
-function removeForestClientFromList(index: number) {
-    forestClientNumberVerifyErrors.value = [];
```

Comment:
removed this line, this method is now at line 193. Because when we remove a forest client number from the information card, there is no need to remove the validation error under the input field. They are two difference places.